### DISCUSSION TITLE
Background jobs/queues

### DISCUSSION THREAD

**Discussion by realprabs** [adonisjs/core#4373] [Ideas]
Pushing everything that is not needed for responding to the current HTTP request to a job queue results in a snappier user experience. Common use cases include sending emails, notifications, syncing data to/from an external service.

Laravel: https://laravel.com/docs/10.x/queues#introduction
Rails: https://edgeguides.rubyonrails.org/active_job_basics.html

Is a first party package for something like this on the roadmap?

**Comment by thetutlage** [comment]
Yes, having an official queue package is needed but not in the roadmap right now. So, the best option is to use bullmq for now as you would do with any other Node.js app.
